### PR TITLE
[API-142] New Notifications Handler

### PR DESCRIPTION
**Pull request by QMalcolm**

### CODE REVIEW

File: web/models/notifications.ex [hunk 1]
```diff
@@ -64,4 +65,27 @@ defmodule Thegm.Notifications do
     |> Repo.update_all(set: [new: false])
   end
 
+  def create_notifications(body, type, recipients, resources, notify_at \\ DateTime.utc_now) when is_bitstring(body) and is_bitstring(type) and is_list(recipients) and is_list(resources) do
+    Enum.each(recipients, fn (x) -> insert_notification(body, type, x, resources, notify_at) end)
```
Comment (SuperManifolds) [inline web/models/notifications.ex]: I'm not a huge fan of variable names like "x" even for arrow functions it makes code hard to read, could we rename it to something like recipient.

File: web/models/notifications.ex [hunk 2]
```diff
@@ -64,4 +65,27 @@ defmodule Thegm.Notifications do
...
+  end
+
+  defp insert_notification(body, type, users_id, resources, notify_at) do
+    notification_changeset = create_changeset(%Thegm.Notifications{}, %{"body" => body, "type" => type, "notify_at" => notify_at, "users_id" => users_id})
+
+    multi = Multi.new
+      |> Multi.insert(:notifications, notification_changeset)
+      |> Multi.run(:notification_resources, fn %{notifications: notification} ->
+        temp = %{notifications_id: notification.id, inserted_at: NaiveDateTime.utc_now, updated_at: NaiveDateTime.utc_now}
+        resources = Enum.reduce(resources, [], fn (x, acc) -> [Map.merge(x, temp) | acc] end)
```
Comment (SuperManifolds) [inline web/models/notifications.ex]: Same as the other comment,

**Review by SuperManifolds — CHANGES_REQUESTED** [review]
Requested changes are commented

**Comment by QMalcolm** [comment]
Fair. I'll hop right on that.

**Review by SuperManifolds — APPROVED** [review]

**Comment by SuperManifolds** [comment]
:party_parrot:

**Pull request merged**